phpunit/fonts-library/wpFontFamily/base.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Test Case for WP_Font_Family tests.
+ *
+ * @package WordPress
+ * @subpackage Fonts Library
+ */
+abstract class WP_Font_Family_UnitTestCase extends WP_UnitTestCase {
+
+	/**
+	 * Fonts directory (in uploads).
+	 *
+	 * @var string
+	 */
+	protected static $fonts_dir = '';
```

Comment:
```suggestion
	protected static $fonts_dir;
```
This is set in `::set_up_before_class()`. Not sure initializing it here is beneficial.